src/log.ts

```diff
@@ -98,6 +98,11 @@ export function debug(..._: any[]) {
 export namespace message {
   export const INVALID_SPEC = 'Invalid spec';
 
+  // FIT
+  export const FIT_NON_SINGLE = 'Autosize fit only works well for single views that are not composed of other views except if layered.';
```

Comment:
'Autosize "fit" only works for single views and layered views'

Reply:
Done